grib2_to_cb/grib_builder_parent.py

```diff
@@ -507,6 +521,7 @@ def build_document(
                         "typeOfLevel": "heightAboveGround",
                         "stepType": "instant",
                         "level": 2,
+                        #"indexpath": "",  # supress the idx files
```

Comment:
You know best in this case. However, I'm generally a fan of removing commented out code if it's not immediately useful. The SOS project ended up with large chunks of commented out code from engineers that had moved on to other projects that no one knew what to do with or if it was still relevant.

Reply:
Actually I feel that way too but this is a placeholder for a problem that I expect is still pending when I run this against data from /public. I fully expect it to fail but that will be in a different PR. I'll be sure to remove it then.

Reply:
Gotcha! That makes sense to me!